packages/components/src/font-size-picker/index.tsx

```diff
@@ -46,61 +99,61 @@ const UnforwardedFontSizePicker = (
 		__next40pxDefaultSize = false,
 		fallbackFontSize,
 		fontSizes = [],
-		disableCustomFontSizes = false,
 		onChange,
+		pickerMode = 'both',
 		size = 'default',
 		units: unitsProp = DEFAULT_UNITS,
 		value,
 		withSlider = false,
 		withReset = true,
+
+		// deprecated
+		disableCustomFontSizes,
 	} = props;
 
-	const units = useCustomUnits( {
-		availableUnits: unitsProp,
-	} );
+	let computedPickerMode = pickerMode;
+	if ( disableCustomFontSizes !== undefined ) {
+		deprecated(
```

Comment:
I would classify this as a [passive deprecation](https://github.com/WordPress/gutenberg/issues/61099), since there is negligible bundle/maintenance cost and no strong reason for a consumer to update their existing `disableCustomFontSizes` usage. What do you think?

Reply:
Sounds good — I removed the `deprecated` warning